jxl/src/features/patches.rs

```diff
@@ -325,6 +462,7 @@ impl PatchesDictionary {
                 positions.push(pos);
             }
 
+            ref_positions.try_reserve(1)?;
```

Comment:
We don't need this because we've already allocated the memory using `new_with_capacity` at line 293.

Reply:
Done. Thanks!